src/databricks/labs/ucx/install.py

```diff
@@ -907,6 +907,75 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def uninstall(self):
+        if not self._prompts or self._question(
+            "Do you want to uninstall ucx from the workspace too, this would "
+            "remove ucx project folder, dashboards, queries and jobs",
+            default="yes",
```

Comment:
```suggestion
            default="no",
```
let's be more cautious :)